indicators/no_critical_vulnerability.json

```
@@ -0,0 +1,24 @@
+{
+    "@context": "https://w3id.org/everse/rsqi#",
+    "name": "noCriticalVulnerability",
+    "description": "Checks if reported critical vulnerabilities has been fixed",
+    "keywords": ["security", "critical", "vulnerability"],
+    "identifier": { "@id": "https://w3id.org/everse/i/indicators/no_critical_vulnerability"},
+    "status": "Active",
+    "version": "1.0.0",
+    "author": {
+        "@type": "schema:Organisation",
```

```suggestion
        "@type": "schema:Organization",
```
Organisation does not exist in schema.org